### PR TITLE
PHP/MySQL Course  and ReactJs Cheatsheet added.

### PR DESCRIPTION
**Pull request by pcabreram1234**
## What does this PR do?
Add resource(s)

### Description

I added one course and one cheatsheet to this amazing repository

### Why is this valuable (or not)?

In my opinion and lived experience this two resources helped me a lot in my web developer beginnings.

### How do we know it's really free?

Easy, The course is from YouTube and the Cheatsheet is from a free website

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-es.md [hunk 1]
```diff
@@ -151,6 +151,7 @@
 * [Clojure](https://programadorwebvalencia.com/cursos/clojure/introducci%C3%B3n/) - Andros Fenollosa (Programador Web Valencia)
 * [Curso C#](https://www.pildorasinformaticas.es/course/curso-c) - Juan Díaz (Píldoras Informáticas)
 * [Curso de JavaScript Gratis](https://codigofacilito.com/cursos/javascript) - Código Facilito
+* [Curso de PHP/MySQL](https://www.youtube.com/watch?v=I75CUdSJifw&list=PLU8oAlHdN5BkinrODGXToK9oPAlnJxmW_) - Juan Díaz (Píldoras informáticas - YouTube)
```
Comment (eshellman) [inline courses/free-courses-es.md:154]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Comment (pcabreram1234) [reply courses/free-courses-es.md]: I just made the changes suggested. Please review it again.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by pcabreram1234** [comment]
Thanks it is very amazing to participate on Open Source Projects.

**Pull request merged**